Makefile

```diff
@@ -24,6 +24,9 @@ YQ_VERSION = v4.41.1
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.26
 
+# OCP Version: for Red Hat bundle community
+OCP_VERSION = 4.12
```

Comment:
when the goal is to not have any version specific values in git, and I would like to keep it that way, I guess this needs to be an input without default value, which is validated and used for the OKD (and OCP?) build only. WDYT?

Reply:
So, this is going to be an input for the workflow as well. 
For OCP is not a problem, because it's called via script (render_template), but not OKD